backend/src/main/java/gov/cdc/usds/simplereport/service/sms/SmsService.java

```diff
@@ -21,7 +21,7 @@
 public class SmsService {
   private static final Logger LOG = LoggerFactory.getLogger(SmsService.class);
 
-  @Value("${twilio.from-number:+14045312484}")
+  @Value("${twilio.from-number:+12023014570}")
```

Comment:
Should we set the default `twilio.from-number` in `application.yaml`? Not sure what the best practice here is.

Reply:
I don't have a preference and I'm also unsure on the best practice 🤷

Reply:
I'm a fan of doing it in the application.yml file; keep all the config details out of code. If we needed to switch out the non-prod `from-number`, it'd make a lot more sense to me to look in application.yml than diving into the app code

Reply:
Done